src/header/Account.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import Button from './Button';
+import AccountEmail from './AccountEmail';
+import AccountImage from './AccountImage';
+import imgSrc from '../img/profile img.svg';
+const url = 'https://bootcamp-api.codeit.kr/api/sample/user';
+ 
+
+function Account(){
+  const [info, setInfo] = useState();
+  const [imgsrc, setImgsrc] = useState();
+  const [login, setLogin] = useState('로그인');
+
+  const handleLogin = () => {
+    fetch(url)
+    .then((response) => response.json())
+    .then((result) => {
+      setInfo(result['email']);
+    })
+      setImgsrc(imgSrc);
+      setLogin('로그아웃')
+  }
+  const handleLogout = () => {
+      setInfo('');
+      setImgsrc('');
+      setLogin('로그인')
+  }
+  return (
+    <div className="Account">
+        <AccountImage value={imgsrc} /> 
+        <AccountEmail content={info}/>
+        <Button text={login} onClick={login==='로그인' ? handleLogin : handleLogout}/>
```

Comment:
```suggestion
  const [isLogin, setIsLogin] = useState(false);

  const handleLogin = () => {
    fetch(url)
    .then((response) => response.json())
    .then((result) => {
      setInfo(result['email']);
    })
      setImgsrc(imgSrc);
      setIsLogin(true);
  }
  const handleLogout = () => {
      setInfo('');
      setImgsrc('');
      setIsLogin(false);
  }
  return (
    <div className="Account">
        <AccountImage value={imgsrc} /> 
        <AccountEmail content={info}/>
        <Button text={isLogin ? '로그인' : '로그아웃'} onClick={isLogin ? handleLogin : handleLogout}/>
```
[중요도: 높음] 개인적으로는 login의 title string 보다는 isLogin(boolean)을 써서 로그인 여부를 검증하고 그 값을 밑에 활용하는게 더 좋아보이네요. 타이틀이 아주 만약에 로그인말고 login으로 바꿔주세요 할 때 관련 코드를 다 바꿔야할 수 있으니까요. 무엇보다 로그인이 된건지 안된건지 더 확실하게 보이니까요

검증은 boolean으로 state로 저장하고, 타이틀은 view에서만 쓰는게 좋아보입니다.

Reply:
수정했습다. 1cea9f27c1448a60713dca83efc26e671a7cf417